### DISCUSSION TITLE
Scale among different axes OR drawing ovals

### DISCUSSION THREAD

**Discussion by NATateo** [cduck/drawSvg#129] [Q&A] [answered]
Hi all
I created a SVG image with a circular color wheel
![circle](https://github.com/cduck/drawsvg/assets/170121725/916fa25d-d004-4bf4-bb1a-72d6f167fc51)
Unfortunately strict limitations in the printable area require the picture to scale differently among X and Y, so that the circle becomes an horizontal oval.
Is it possible?

How would you draw the same picture with an oval shape rather than circular?

**Comment by cduck** [comment, marked as answer]
You can use the Ellipse element to draw ellipses or you can all your shapes inside a Group element with [the attribute `transform="scale(1,1.4)"`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform).

**Reply by NATateo** [reply]
Thank you, that's exactly what I was looking for
